elasticsearch/plugins.sls

```diff
@@ -1,17 +1,11 @@
 include:
   - elasticsearch.pkg
 
-{%- if salt['pillar.get']('elasticsearch:plugins', {}) %}
-
-{% for name, repo in elasticsearch.plugins.iteritems() %}
```

Comment:
There is no need for the ``if`` condition since the default value is set to ``{}``.